crates/biome_cli/tests/snapshots/main_commands_check/check_help.snap

```diff
@@ -77,6 +77,10 @@ Global options applied to all commands
                               [default: none]
         --log-kind=<pretty|compact|json>  How the log should look like.
                               [default: pretty]
+        --diagnostic-level=<info|warn|error>  The level of logging. In order, from the most verbose to
```

Comment:
You might have noticed that the documentation is wrong. I think you copied the one from the `--log-level` and forgot to update it.

Reply:
Yes that's kind of what happened, I even updated it, just forgot to commit it.

On the topic of the default being wrong here, if we want users to be able to specify "info", we need to parse that to `Severity::Information`, which creates a bit of inconsistency between how it needs to be specified in the CLI and how it's represented internally. Same thing applies to the `warn` and `Severity::Warning`. Is that okay to do in your opinion? If so, then I'll make the modifications to the `FromStr` and `Display` traits.

Reply:
Yeah you're right, we should update those too to reflect the values we accept via CLI

Reply:
Sorry, from this comment it's not 100% clear to me what exactly I should update. Do we want the members of the `Severity` enum refactored to `Warn` and `Info` respectively or just update the parsing so `info` gets parsed to `Severity::Information` and `warn` gets parsed into `Severity::Warning`?

Reply:
Yeah sorry, I think I misunderstood the initial message. What we want is something like the following:

```rs
impl FromStr for Severity {
    type Err = String;

    fn from_str(s: &str) -> Result<Self, Self::Err> {
        match s {
            "hint" => Ok(Self::Information),
            "info" => Ok(Self::Information),
            "warn" => Ok(Self::Warning),
            "error" => Ok(Self::Error),
            v => Err(format!("Found unexpected value ({}), valid values are: info, warn, error.", v)),
        }
    }
}
```

Hint and info, **for now** will have to map to the same variant, and we don't want users to have control over fatal/error messages. So errors and fatal diagnostics must always be printed, and users can't mute errors.

Reply:
Thanks for the clarification!